README.md

```diff
@@ -45,7 +45,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: joergbrech/moxunit-action@v1
+      - uses: joergbrech/moxunit-action@v1.2.0
```

Comment:
The tag v1 actually always points to the latest version 1.x.x, which currently is 1.2.0. So using v1 is the "more stable" choice and I would like to motivate this in the README.